initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -217,7 +217,9 @@
                     monitorinterval="900"
                     monitorqueue="dfuserver_monitor_queue"
                     name="mydfuserver"
-                    queue="dfuserver_queue">
+                    queue="dfuserver_queue"
+                    noCommon="true"
+                    maxConnections="-1">
```

Comment:
See other comment, should be in a global section of the environment.xml..

Reply:
Moved